tools/functional-tester/etcd-tester/tester.go

```diff
@@ -83,6 +83,13 @@ func (tt *tester) runLoop() {
 			plog.Printf("[round#%d case#%d] recovered failure", i, j)
 
 			if !tt.consistencyCheck {
+				plog.Printf("[round#%d case#%d] restarting the stressers...", i, j)
```

Comment:
We return from stresser if there's any error other than `context deadline exceeded`. And we got `too slow: no commit` error which means stresser was not running. So my guess is the stresser got exited from previous rounds when we kill one of the node and we never restart that stresser, therefore we ran out of stressers triggering `too slow` error. So this restarts all stressers because we don't know which stressers got exited.

I still need to confirm that that was this case adding more logs to this branch.
I will let you know when I find out why.

Thanks.